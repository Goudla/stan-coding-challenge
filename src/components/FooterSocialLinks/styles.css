.footer-social-links {
  flex: 1;
  margin-top: 47px;
}
