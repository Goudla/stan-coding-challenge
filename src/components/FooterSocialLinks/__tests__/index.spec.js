import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';

// Scenes
import FooterSocialLinks from '../';

describe('<FooterSocialLinks />', () => {
  it('Should render', () => {
    const renderer = new ShallowRenderer();
    renderer.render(<FooterSocialLinks>Test</FooterSocialLinks>);
    const result = renderer.getRenderOutput();
    expect(result).toMatchSnapshot();
  });
});
