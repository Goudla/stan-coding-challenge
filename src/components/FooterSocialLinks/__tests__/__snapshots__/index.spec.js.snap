// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<FooterSocialLinks /> Should render 1`] = `
<div
  className="footer-social-links"
>
  Test
</div>
`;
